docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -818,6 +818,12 @@ $ curl localhost:8888/encrypt -d mysecret
 NOTE: If the value you are encrypting has characters in it that need to be URL encoded you should use
 the `--data-urlencode` option to `curl` to make sure they are encoded properly.
 
+TIP: The encrypted output when shown on console is obfuscated by adjacent 'Current Speed' value, redirecting the output to file helps avoid this confusion.
```

Comment:
Maybe we should make this more general, specifically calling out Current Speed might confuse people.  I think if you just said something like 
"Be sure not to include any of the `curl` command statistics in the encrypted value. 
 Outputting the value to a file can help avoid this problem." 
that would suffice.